common/src/primitives/id/mod.rs

```diff
@@ -122,6 +122,12 @@ impl<T: ?Sized> Id<T> {
     }
 }
 
+impl<T: Debug> Debug for Id<T> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        std::fmt::Debug::fmt(&self.id, f)
```

Comment:
I think we need a separate PR because the current format of printing is quite useless and there are things to be changed here. I changed them here because this is the format that made sense while debugging. Let's have a discussion about this in the other PR.